src/constants/poi.data.ts

```diff
@@ -30,6 +30,18 @@ export const POIToSearchResult = (poi: POI): SearchResult => {
   };
 };
 
+export const MarkerToSearchResult = (
+  markerLocation: MarkerLocation
+): SearchResult => {
+  return {
+    id: markerLocation.id,
+    displayName: markerLocation.displayName,
+    location: markerLocation.location,
+    searchName: markerLocation.displayName,
+    extraInformation: "",
+  };
+};
+
```

Comment:
Please remove this as it was already moved to UtilityService

Reply:
Done.